webapp/pages/settings/index.vue

```diff
@@ -135,15 +135,12 @@ export default {
             })
           }
         })
-        .then(data => {
-          this.$toast.success('Updated user')
-        })
-        .catch(err => {
-          this.$toast.error(err.message)
-        })
-        .finally(() => {
-          this.sending = false
-        })
+        this.$toast.success(this.$t('settings.data.success'))
+      } catch (err) {
+        this.$toast.error(err.message)
+      } finally {
+        this.loadingData = false
```

Comment:
```suggestion
        this.pending = false
```
I believe `pending` is already a convention in this project.

Reply:
![image](https://user-images.githubusercontent.com/1238238/56419523-c587ae80-629a-11e9-936f-f3ea310ce267.png)

Operation name in template is `loading`

Suggestion: No Triage

Reply:
Yes, `loading` is good if the component expects a property with that name.

Reply:
What do you mean by

> Suggestion: No Triage

by the way?

Reply:
`no triage` means that there will be no fix for the raised issue. It is a medical term and would translate to smth like `keine Behandlung` (German). It is used with google repos.
If I `Suggestion: No Triage`, I present an argument why I don't want to fix this and ask for approval of this course of action.